go-gitea/gitea
# Prevent re-review and dismiss review actions on closed and merged PRs

Resolves #29965.

---
Manually tested this by:
- Following the [installation](https://docs.gitea.com/next/installation/install-with-docker#basics) guide (but built a local Docker image instead)
- Creating 2 users, one who is the `Owner` of a newly-created repository and the other a `Collaborator`
- Had the `Collaborator` create a PR that the `Owner` reviews
- `Collaborator` resolves conversation and `Owner` merges PR

And with this change we see that we can no longer see re-request review button for the `Owner`:

<img width="1351" alt="Screenshot 2024-03-25 at 12 39 18 AM" src="https://github.com/go-gitea/gitea/assets/60799661/bcd9c579-3cf7-474f-a51e-b436fe1a39a4">
